application/views/questionAdministration/subquestionRow.twig

```diff
@@ -129,18 +129,21 @@
         </div>
     </td>
 
+    {% set relevanceTooltip = supportsRelevance ? gT("Click to expand") : gT("Relevance is not currently supported for columns") %}
+
     <!-- Relevance equation -->
     {% if first and scale_id == 0 %}
```

Comment:
`{% if supportsRelevance and first and scale_id == 0 %} ` and same for header ?
(best solution)